netlink/uevent.go

```diff
@@ -19,6 +21,8 @@ const (
 	UNBIND  KObjAction = "unbind"
 )
 
+const libudevMagic = 0xfeedcafe
```

Comment:
Add a comment to udev src, ie: https://github.com/systemd/systemd/blob/v239/src/libudev/libudev-monitor.c#L57

Reply:
Added.